airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/build_image/steps/common.py

```diff
@@ -76,7 +78,8 @@ async def _build_connector(self, platform: Platform, *args: Any, **kwargs: Any)
         """
         raise NotImplementedError("`BuildConnectorImagesBase`s must define a '_build_connector' attribute.")
 
-    async def _get_image_user(self, base_container: Container) -> str:
+    @classmethod
+    async def get_image_user(cls: Type[T], base_container: Container) -> str:
```

Comment:
Making it a classmethod so that it can be called without step instance - as a kind of util.

Reply:
Nice! I like that this sets a clear contract for the user attribute across instances.